packages/docs/components/tab.md

```diff
@@ -25,16 +22,312 @@ links:
 
 ## Anatomy
 
+
+<Anatomy img="/images/anatomy-tab.svg" />
+
+## Behavior
+
 <Description>
 
-<span class="is-fpo">Descriptive content around **tab anatomy** should go here.</span>
+Users interact with Tab as they would a button, the main difference is the outcome. By default, the first tab from the left is active and the associated content 
+displayed in the tab panel. Upon selecting a different tab, the tab indicator appears on the selected tab. The tab panel will then update to show the new content. 
+
+Additionally, the tab element is keyboard-navigable (See [Accessibility: Keyboard Support](#keyboard-support))
 
 </Description>
 
-<Anatomy img="/images/anatomy-tab.svg" />
+<Visual layout="wide" content="full" fade>
+  <template>
+    <header>
+      <h1>Terrestrial Planets</h1>
+      <p>Terrestrial planets are planets that are composed primarily of silicate rocks or metals.</p>
+    </header>
+    <OdsTabs aria-label="Types of terrestrial planets" :active="tabsPlanets.active" :tablist="tabsPlanets.tablist" id="example-1">
+      <template slot="tab-mercury">
+        <blockquote class="is-sample-unimportant">
+          <p>Mercury is the smallest and innermost planet in the Solar System. Its orbit around the Sun takes 87.97 Earth days, the shortest of all the planets in the Solar System.</p>
+        </blockquote>
+      </template>
+      <template slot="tab-venus">
+        <blockquote class="is-sample-unimportant">
+          <p>Venus is the second planet from the Sun. It is named after the Roman goddess of love and beauty. As the second-brightest natural object in the night sky after the Moon, Venus can cast shadows and can be, on rare occasion, visible to the naked eye in broad daylight.</p>
+        </blockquote>
+      </template>
+      <template slot="tab-earth">
+        <blockquote class="is-sample-unimportant">
+          <p>Earth is the third planet from the Sun and the only astronomical object known to harbor life. About 29% of Earth's surface is land consisting of continents and islands.</p>
+        </blockquote>
+      </template>
+      <template slot="tab-mars">
+        <blockquote class="is-sample-unimportant">
+          <p>Mars is the fourth planet from the Sun and the second-smallest planet in the Solar System, being larger than only Mercury. In English, Mars carries the name of the Roman god of war and is often referred to as the "Red Planet".</p>
+        </blockquote>
+      </template>
+    </OdsTabs>
+  </template>
+</Visual>
+
+## Usage
+
+<Description>
+
+Tabs were created to shorten long pages. Before you use these, we’d recommend laying 
+out all of the content on your page out first. From there figure out common themes and 
+see what could be grouped together. Those themes should become your Tabs.
+
+</Description>
+
+### Relate to the page title
+
+<Description>
+
+Tabs are NOT navigation. Meaning they don’t take you from place to place. Rather, they are meant for context switching related to the page.
```

Comment:
I'd prefer <strong> or <em> to caps here

Reply:
Agreed. Fixed.